### DISCUSSION TITLE
API access?

### DISCUSSION THREAD

**Discussion by ljharb** [community/OSPO#60] [Dashboard]
Will this data eventually be available in the API? I'd love to be able to work with it programmatically.

**Comment by ahpook** [comment]
Yeah we're definitely looking at making this accessible via API. Can you say a bit more about what you'd like to accomplish? Do you have a preference for graphql or REST?

**Reply by ljharb** [reply]
I mean, ideally everything would be in both, but the graphql API is what i'm using most often these days.

In general, literally *any task* I can do on the website, I want to be able to automate. In this case, I'd want to be able to automate multi-org/multi-user management, surfacing repos that need attention based on whatever metrics I'm interested in at that time, in the place I'm most equipped to take action - the command line.

**Reply by Lestad77** [reply]
Available to learn
